client/components/jetpack/wpcom-business-at/index.tsx

```diff
@@ -0,0 +1,157 @@
+import useTrackCallback from 'lib/jetpack/use-track-callback';
+import { getAutomatedTransferStatus } from 'state/automated-transfer/selectors';
+import { initiateThemeTransfer } from 'state/themes/actions';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { transferStates } from 'state/automated-transfer/constants';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Asset dependencies
+ */
+import JetpackBackupSVG from 'assets/images/illustrations/jetpack-backup.svg';
+import JetpackScanSVG from 'assets/images/illustrations/jetpack-scan.svg';
+
+interface Props {
+	product: 'backup' | 'scan';
+}
+
+const contentPerPrimaryProduct = {
+	backup: {
+		documentHeadTitle: 'Activate Jetpack Backup now',
+		header: translate( 'Jetpack Backup' ),
+		primaryPromo: {
+			title: translate( 'Get time travel for your site with Jetpack Backup' ),
+			image: { path: JetpackBackupSVG },
+			content: translate(
+				'Backup gives you granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
+			),
+			promoCTA: {
+				text: translate( 'Activate Jetpack Backup now' ),
+				loadingText: translate( 'Activating Jetpack Backup' ),
+			},
+		},
+		secondaryPromo: {
+			title: translate( 'Jetpack Scan' ),
+			icon: 'security',
+			content: translate(
+				'Automated scanning and one-click fixes to keep your site ahead of security threats.'
+			),
+		},
+	},
+
+	scan: {
+		documentHeadTitle: 'Activate Jetpack Scan now',
+		header: translate( 'Jetpack Scan' ),
+		primaryPromo: {
+			title: translate( 'We guard your site. You run your business.' ),
+			image: { path: JetpackScanSVG },
+			content: translate(
+				'Scan gives you automated scanning and one-click fixes to keep your site ahead of security threats.'
+			),
+			promoCTA: {
+				text: translate( 'Activate Jetpack Scan now' ),
+				loadingText: translate( 'Activating Jetpack Scan' ),
+			},
+		},
+		secondaryPromo: {
+			title: translate( 'Jetpack Backup' ),
+			icon: 'security',
+			content: translate(
+				'Backup gives you granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
+			),
+		},
+	},
+};
+
+export default function WPCOMBusinessAT( { product }: Props ): ReactElement {
+	const content = React.useMemo( () => contentPerPrimaryProduct[ product ], [ product ] );
+	const siteId = useSelector( getSelectedSiteId );
+	const dispatch = useDispatch();
+	const initiateAT = React.useCallback( () => {
+		dispatch( initiateThemeTransfer( siteId, null, null ) );
+	}, [ dispatch, siteId ] );
+	const eventName =
+		product === 'backup'
+			? 'calypso_jetpack_backup_business_at'
+			: 'calypso_jetpack_scan_business_at';
+	const trackInitiateAT = useTrackCallback( initiateAT, eventName );
+
+	const automatedTransferStatus = useSelector( ( state ) =>
+		getAutomatedTransferStatus( state, siteId )
+	);
+
+	return (
+		<Main className="wpcom-business-at">
```

Comment:
@keoshi should we use here `is-wide-layout` like the other Upsell pages (free to business) and also Earn section

Reply:
I think it should be fine _not_ to use it. This means we'll have 1:1 parity with the cloud and ensure it's a future-proof solution going forward, even if that means a slightly narrowed layout in dotcom.